### DISCUSSION TITLE
"not found" error occurred during compiling

### DISCUSSION THREAD

**Discussion by XXJShun** [libMesh/libmesh#3393] [Q&A]
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/libtool: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/libtool: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
/bin/sh: 1: /root/projects/moose/libmesh/installed/contrib/bin/libmesh-config: not found
Building and linking hit...
bash: 行 1: -std=c++17：未找到命令
make: *** [/root/projects/moose/framework/moose.mk:116：hit] 错误 127
Traceback (most recent call last):
  File "/root/projects/moose/python/pyhit/__init__.py", line 13, in <module>
    import hit
ModuleNotFoundError: No module named 'hit'
(base) root@xxjinshun:~/projects/neals#
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/projects/moose/python/pyhit/__init__.py", line 18, in <module>
    import hit
ModuleNotFoundError: No module named 'hit'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/projects/neals/./run_tests", line 11, in <module>
    from TestHarness import TestHarness
  File "/root/projects/moose/python/TestHarness/__init__.py", line 15, in <module>
    from .TestHarness import TestHarness
  File "/root/projects/moose/python/TestHarness/TestHarness.py", line 20, in <module>
    from FactorySystem.Factory import Factory
  File "/root/projects/moose/python/FactorySystem/__init__.py", line 13, in <module>
    from .Parser import Parser
  File "/root/projects/moose/python/FactorySystem/Parser.py", line 13, in <module>
    import pyhit
  File "/root/projects/moose/python/pyhit/__init__.py", line 22, in <module>
    import hit
ModuleNotFoundError: No module named 'hit'

**Comment by roystgnr** [comment]
Looks like you're running `make` in MOOSE, but you haven't built the libMesh submodule there and you haven't set a `$LIBMESH_DIR` to a libMesh install elsewhere.

**Reply by XXJShun** [reply]
Thanks for your answer, I downloaded the libmesh folder from GitHub, but I don't quite understand what you mean by you haven't set a $LIBMESH_DIR to a libMesh install elsewhere.
